buildspecs/aix_ppc-64.spec

```diff
@@ -230,6 +230,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="opt_jxeLoadSupport" value="true"/>
 		<flag id="opt_memoryCheckSupport" value="true"/>
 		<flag id="opt_multiVm" value="true"/>
+		<flag id="opt_ojdkThreadSupport" value="false"/>
```

Comment:
recently we have been adding new 'features' rather than 'specs'. This way you only add it once, rather than in each spec file. In your extensions repo changes, you need to use `FEATURE_SED_SCRIPT` instead of `SPEC_SED_SCRIPT`

Reply:
Switching to `FEATURE_SED_SCRIPT` at https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/387, and removed the spec file changes.

Reply:
I think the flag should still be added to j9.flags.

Reply:
Yeah, fixed it.